android/app/src/main/java/com/pillarproject/wallet/MainApplication.java

```diff
@@ -60,6 +61,11 @@ public void onCreate() {
         // react-native-async-storage custom max storage
         long storageSizeMax = 60L * 1024L * 1024L; // 60 MB
         com.reactnativecommunity.asyncstorage.ReactDatabaseSupplier.getInstance(getApplicationContext()).setMaximumSize(storageSizeMax);
+
+        new RNInstabugReactnativePackage
+            .Builder(getString(R.string.instabug_token), MainApplication.this)
+            .setInvocationEvent("shake")
```

Comment:
Consider adding menu option for invoking instabug in a discoverable way. Shake gesture seems rather had to find for regular users.

Reply:
@mdjastrzebski please see my comment above.